rezervo/chains/sats.py

```diff
@@ -0,0 +1,18 @@
+from rezervo.chains.chain import Chain
+from rezervo.providers.sats.provider import SatsProvider
+from rezervo.providers.schema import Branch, Location
+
+
+class SatsChain(Chain, SatsProvider):
+    identifier = "sats"
+    name = "Sats"
+    branches = [
+        Branch(
+            identifier="oslo/akershus",
+            name="Oslo/Akershus",
+            locations=[
+                Location(identifier="kolbotn", name="Kolbotn", provider_identifier=224),
+                Location(identifier="ryen", name="Ryen", provider_identifier=219),
+            ],
+        ),
+    ]
```

Comment:
I opted to add a couple of semi-random clubs for testing, but we might as well add more. Here is the full list of clubs and corresponding IDs, from the mobile app API.
[clubs.json](https://github.com/mathiazom/rezervo/files/14337413/clubs.json)

Reply:
The frontend is not currently great at handling a very large number of classes, so I would keep the number of locations somewhat low. This highly depends on the number of classes per location, but maybe not more than what we currently have for 3T.

Reply:
Yeah, sure. I added some more in https://github.com/mathiazom/rezervo/pull/43/commits/ee4da08045da5e386441b0c7d1764e9f224719ff

Reply:
Ouf this led me to discover that they have some form of pagination. Unresolved for now

Reply:
Addressed in https://github.com/mathiazom/rezervo/pull/43/commits/d630add56fe16b8eb42b33fbf2fd6f636dfccfe2